src/assets/i18n/en.json5

```diff
@@ -5408,19 +5408,15 @@
 
   "access-control-bitstream-toggle.disable": "Disable option to perform changes on the bitstreams",
 
-  "access-control-mode": "Mode",
+  "search.filters.operator.notequals.text": "Not Equals",
 
-  "access-control-access-conditions": "Access conditions",
+  "search.filters.operator.notcontains.text": "Not Contains",
 
-  "access-control-no-access-conditions-warning-message": "Currently, no access conditions are specified below. If executed, this will replace the current access conditions with the default access conditions inherited from the owning collection.",
+  "search.filters.operator.contains.text": "Contains",
 
-  "access-control-replace-all": "Replace access conditions",
+  "search.filters.filter.title.text": "Title",
 
-  "access-control-add-to-existing": "Add to existing ones",
-
-  "access-control-limit-to-specific": "Limit the changes to specific bitstreams",
-
-  "access-control-process-all-bitstreams": "Update all the bitstreams in the item",
+  "search.filters.filter.author.text": "Author",
```

Comment:
It appears you are missing `search.fitlers.filter.subject.text` in this file.  It needs to be added.